### PR TITLE
feat: support a few domains (404)

### PR DESCRIPTION
**Pull request by nightnei**

### CODE REVIEW

**Review by StyleT — APPROVED** [review]

**Comment by StyleT** [comment]
Pls write E2E tests for this

**Review by StyleT — CHANGES_REQUESTED** [review]
Pls write E2E tests for this

**Comment by nightnei** [comment]
> Pls write E2E tests for this

done

File: e2e/spec/404.spec.e2e.ts [hunk 1]
```diff
@@ -1,32 +1,49 @@
...
     I.waitForElement(notFoundPageLink, 30);
     I.click(notFoundPageLink);
-    I.waitForText('404 not found', 10, 'body > div#body');
+    I.waitForElement('body > div#navbar > div.app-container > div.navbar-app', 30);
+    I.waitForText(common.text404NotFound, 10, 'body > div#body');
+});
+
+Scenario('Renders (SSR) global 404 page for non-existing ILC route for domain "127.0.0.1:8233"', (I, common: common) => {
+    I.amOnPage(common.url.localhostAsIPv4 + '/nonexistent-path');
+    I.waitForText(common.text404NotFound, 10, 'body > div#navbar'); // for "127.0.0.1:8233" inside of navbar will be rendered 404 fragment, like in body
```
Comment (StyleT) [inline e2e/spec/404.spec.e2e.ts:21]: I think this test checks nothing... We need to validate that "other" 404 page can be rendered for custom domain.

**Review by StyleT — APPROVED** [review]

**Pull request merged**